docs/resources/v2-conformance-statement.md

```diff
@@ -0,0 +1,818 @@
+# Version 2 DICOM Conformance Statement
```

Comment:
Is our plan to delete the v1 conformance after we enable v2 for all? Or follow the mandatory 6 months until then we update both.

Reply:
I think preview is 6 months. Released versions have a different lifecycle. There is no huge cost to supporting both. We may have to wait for another release before we start to deprecate V1. I will learn more as I onboard to Azure Advisory